tests/test_fe_utils.py

```diff
@@ -32,3 +34,27 @@ def test_extract_delta_Us_from_U_knk():
     )
 
     np.testing.assert_almost_equal(expected_delta_Us, test_delta_Us)
+
+
+def test_image_molecules():
```

Comment:
I would definitely bullet-proof this test a bit:

1) Use non-cube boxes, i.e. have diag of box be different
2) Also add a test case with a real molecule (whose center has been shifted to outside of the box.

``` python
mol = # some mol from tests folder with 3D coordinates
# ensure initial starting coordinates are somewhere in the box


# randomly modify coordinates by
for _ in range(1000):

   # shift each direction randomly, we may need to go beyond one image since waters can float very far away
   x_shift = np.rand.randint(-5, 5)
   y_shift = np.rand.randint(-5, 5)
   z_shift = np.rand.randint(-5, 5)
   
   offset = np.array([x_shift*box_size_0, y_shift*box_size_1, z_shift*box_size_2])
   offset = np.expand_dims(offset, axis=0) # make this broad castable to [N,3]
   new_mol_conf = old_conf + offset

   imaged_mol = image_molecule(new_mol_conf, box)
   # assert image_mol has same coordinates as imaged_mol

```